compiler/optimizer/OMROptimizer_inlines.hpp

```diff
@@ -1,5 +1,9 @@
 /*******************************************************************************
+<<<<<<< HEAD
  * Copyright (c) 2021, 2021 IBM Corp. and others
+=======
+ * Copyright (c) 2000, 2021 IBM Corp. and others
+>>>>>>> 3f38590e3 (Split inlined OMR::Optimizer::self from cpp file)
```

Comment:
Can you clean these up please?  The merge conflict annotations should not be part of the file you commit.